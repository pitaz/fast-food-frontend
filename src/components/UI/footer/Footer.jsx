import React, { Component } from "react";

class Footer extends Component {
  render() {
    return (
      <footer>
        <p>Designed by Peter Odekwo</p>
      </footer>
    );
  }
}

export default Footer;
