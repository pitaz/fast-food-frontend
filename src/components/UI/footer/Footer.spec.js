/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from './Footer.jsx';

describe('Footer component test', () => {
  const wrapper = shallow(<Footer />);
  it('renders the Footer component', () => {
    expect(wrapper.exists()).toBe(true);
  });
  it('should match the snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
