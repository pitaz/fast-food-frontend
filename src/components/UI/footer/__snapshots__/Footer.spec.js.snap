// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component test should match the snapshot 1`] = `
<footer>
  <p>
    Designed by Peter Odekwo
  </p>
</footer>
`;
